evap/staff/templates/staff_evaluation_person_management.html

```diff
@@ -15,7 +15,9 @@
                     <div class="card-body h-100">
                         <h4 class="card-title">{% trans 'Import participants' %}</h4>
                         <h6 class="card-subtitle mb-2 text-muted">{% trans 'From Excel file' %}</h6>
-                        <p class="card-text">{% trans 'Upload Excel file with participant data' %} (<a href="{% url 'staff:download_sample_xls' 'sample_user.xls' %}">{% trans 'sample file' %}</a>). {% trans 'This will create all containing users.' %}</p>
+                        <p class="card-text">{% trans 'Upload Excel file with participant data' %} (<a href="{% url 'staff:download_sample_xls' 'sample_user.xls' %}">{% trans 'sample file' %}</a>,
+                        <a href="#" onclick="copyHeaders(['User name', 'Title', 'First name', 'Last name', 'Email'])">{% trans 'copy sheet headers' %}</a>).
+                        {% trans 'This will create all containing users.' %}</p>
```

Comment:
I think this has now become enough code to indent it. Probably
```django
<p class="card-text">
    {% trans 'Upload Excel file with participant data' %}
    (<a href="{% url 'staff:download_sample_xls' 'sample_user.xls' %}">{% trans 'sample file' %}</a>,
    <a href="#" onclick="copyHeaders(['User name', 'Title', 'First name', 'Last name', 'Email'])"> {% trans 'copy sheet headers' %}</a>).
    {% trans 'This will create all containing users.' %}
</p>
```
is a bit more readable.

Reply:
Same applies to the other places where you added this.